lib/internal/streams/end-of-stream.js

```diff
@@ -255,6 +263,15 @@ function eos(stream, options, callback) {
   return cleanup;
 }
 
+function eosWeb(stream, opts, callback) {
```

Comment:
This needs to use the brand checks, so that it can throw `ERR_INVALID_ARG_TYPE` if `stream` is not a valid web stream type.

Reply:
Done have added checks for non-streams